### PR TITLE
MyRocks landing page

### PR DESCRIPTION
**Pull request by zardra**
Would really like to have the Installation and Migration sections reviewed to ensure they are correct technically.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 permalink: /blog/:year/:month/:day/:title.html
-title: ProjectName
-tagline: This is a brief one-liner about the project
+title: MyRocks
+tagline: A RocksDB storage engine with MySQL
 description: >
-  This is a longer description of the project, a bit of a sales-pitch with a bit of technical information included.
+  Benefit from all the features of MySQL while using RocksDB as backend storage
 fbappid: "1615782811974223"
 gacode: "UA-12345-6"
```
Comment (JoelMarcey) [inline _config.yml]: We are going to need to add Google Analytics for this.

File: _config.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ ghrepo: "facebook/Site-Templates"
 # Use these color settings to determine your colour scheme for the site.
 color:
   # primary should be a vivid color that reflects the project's brand
-  primary: "#3b5998"
+  primary: "#0f2647"
```
Comment (JoelMarcey) [inline _config.yml:38]: Ugh -- can't make a comment on line 21 :/

Anyway, for `url`, it should be:

`url: http://facebook.github.io/`

and on line 15, you should add a `baseurl` that is:

`baseurl: mysql-5.6`

Comment (JoelMarcey) [inline _config.yml:38]: ...and line 32 should be changed to `facebook/mysql-5.6` for the repo

File: static/logo1.svg [hunk 3]
```diff
@@ -0,0 +1,560 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (JoelMarcey) [inline static/logo1.svg]: What is `logo1.svg`?

Comment (zardra) [reply static/logo1.svg]: I think that's from me messing around with images.  Easy enough to delete.

File: _config.yml (hunk 2 above)
Comment (zardra) [reply _config.yml:38]: Are these suppose to have quotes? The example in the file does...

Comment (zardra) [reply _config.yml:38]: Done

File: _config.yml (hunk 1 above)
Comment (zardra) [reply _config.yml]: Is there something special I need to do for that?

And what about the fbappid?

Comment (JoelMarcey) [reply _config.yml]: We can do the GA id together. I will need to add something to our internal GA page, I think.

I believe the fbappid stays the same across all things using this template.

**Pull request merged**